frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -151,16 +135,34 @@ function ReaderPaging:setupTapTouchZones()
     })
 end
 
--- This method will be called in onSetDimensions handler
 function ReaderPaging:setupTouchZones()
-    -- deligate gesture listener to readerui
```

Comment:
I hope you removed these comments because they are wrong/obsolete - and not because you don't see them in the other modules or because they are vague (I'd rather see vague comments in one module than none at all, it might still give a hint at what's happening to the user looking around - I remember I did when discovering the gesture handle infrastructure - but I admit these comments look not good enough :)

Reply:
Is it for future purposes? Because there is no onSetDimensions handler in ReaderPaging (there is in ReaderRolling).

Reply:
Well, I had to look at them - and I think they are indeed wrong/obsolete. So, good to remove them.

And no, no future purpose.
Only the `-- delegate gesture listener to readerui` gives a hint about what this is about (although it's documented in inputcontainer.lua, but this sentence in at least one of the ReaderModule does not hurt - it is also as-is in ReaderMenu and ReaderHighlight, so good).